package.json

```diff
@@ -21,38 +21,55 @@
     "analyze": "webpack --profile --json --progress > stats.json",
     "check-types": "tsc --noEmit",
     "lint": "eslint . --ext js --ext jsx --ext ts --ext tsx --cache",
-    "fix": "npm run lint -- --fix"
+    "fix": "npm run lint -- --fix",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "jest": {
```

Comment:
We're already using separate files for our other configs (`.babelrc`, `.eslintrc`, `.stylelintrc`, `tsconfig.json`). Would you mind creating a `jest.config.js` file?